ssc/cmod_pvsamv1.cpp

```diff
@@ -1128,7 +1132,7 @@ void cm_pvsamv1::exec( ) throw (general_error)
 			// if PV simulation is subhourly.  load is assumed constant over the hour.
 			// if no load profile supplied, load = 0
 			if (nload == 8760)
-				cur_load = p_load_in[hour];
+				cur_load = p_load_in[hour] * load_scale[iyear];
```

Comment:
It looks like `cur_load` is being calculated with scaling and then stored in the lifetime vector `p_load_full`. Could  `single_year_to_lifetime_interpolated` be used from the start like in other cmods?

Reply:
Thanks for the suggestion! Looks like cur_load was used in some commented out logging functions. I removed it and replaced it with something that matches the other cmods in the latest commit.